opensrp-chw/src/androidTest/java/org/smartregister/chw/activity/wcaro/FamilyMemberTest.java

```diff
@@ -0,0 +1,197 @@
+        onView(withSubstring("SAVE"))
+                .perform(click());
+    }
+
+    @Test
+    @Order(order = 4)
+    public void changePrimarycareGiverSuccessfully() throws InterruptedException{
+        onView(withHint("Search name or ID"))
+                .perform(typeText(Configs.TestConfigHelper.familyName), closeSoftKeyboard());
+        onView(withSubstring(Configs.TestConfigHelper.familyName + " Family"))
+                .perform(click());
+        Thread.sleep(500);
+        utils.openFamilyMenu();
+        Thread.sleep(500);
+        onView(withSubstring("Change primary caregiver"))
+                .perform(click());
+        Thread.sleep(500);
+        onView(withSubstring(Configs.TestConfigHelper.aboveFiveFirstNameTwo
+                + " " + Configs.TestConfigHelper.aboveFiveSecondNameTwo + " " + Configs.TestConfigHelper.familyName + ", "
+                + Configs.TestConfigHelper.aboveFiveage))
+                .perform(click());
+        Thread.sleep(2000);
+        onView(withHint("Phone number")).perform(clearText(), typeText(Configs.TestConfigHelper.phoneNumberOne));
+        onView(withHint("Other phone number"))
+                .perform(clearText(), typeText(Configs.TestConfigHelper.getPhoneNumberTwo));
+        onView(withSubstring("SAVE"))
+                .perform(click());
+    }
+
+    @Test
+    @Order(order = 1)
+    public void confirmFamilyHead() throws InterruptedException {
+        onView(withHint("Search name or ID"))
+                .perform(typeText(Configs.TestConfigHelper.familyName), closeSoftKeyboard());
+        onView(withSubstring(Configs.TestConfigHelper.familyName + " Family"))
+                .perform(click());
+        Thread.sleep(500);
+        onView(withSubstring(Configs.TestConfigHelper.aboveFiveFirstNameOne
+                + " " + Configs.TestConfigHelper.aboveFiveSecondNameOne + " " + Configs.TestConfigHelper.familyName + ", "
+                + Configs.TestConfigHelper.aboveFiveage))
+                .perform(click());
+        onView(withId(R.id.family_head))
+    .check(matches(isDisplayed()));
+    }
+
+    @Test
+    @Order(order = 2)
+    public void confirmUniqueId() throws InterruptedException {
+        onView(withHint("Search name or ID"))
+                .perform(typeText(Configs.TestConfigHelper.familyName), closeSoftKeyboard());
+        onView(withSubstring(Configs.TestConfigHelper.familyName + " Family"))
+                .perform(click());
+        Thread.sleep(500);
+        onView(withSubstring(Configs.TestConfigHelper.aboveFiveFirstNameOne
+                + " " + Configs.TestConfigHelper.aboveFiveSecondNameOne + " " + Configs.TestConfigHelper.familyName + ", "
+                + Configs.TestConfigHelper.aboveFiveage))
+                .perform(click());
+        onView(withId(R.id.textview_detail_three))
+                .check(matches(isDisplayed()));
+    }
+
+    @Test
+    @Order(order = 3)
+    public void confirmFabOptions() throws InterruptedException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5426621)